src/components/LogoutButton/LogoutButtonComponent.js

```diff
@@ -1,12 +1,15 @@
 import React, { Component } from 'react';
-import { StyledLogoutButton } from 'components/LogoutButton/LogoutButtonStyles';
+// import { StyledLogoutButton } from 'components/LogoutButton/LogoutButtonStyles';
```

Comment:
I don't see anything useful inside `LogoutButtonStyles.js`. Let's delete it and let's delete this commented code.

Reply:
Please leave that to refactoring branch